pkg/ccl/streamingccl/streamproducer/producer_job.go

```diff
@@ -44,12 +44,17 @@ func makeProducerJobRecord(
 	timeout time.Duration,
 	user username.SQLUsername,
 	ptsID uuid.UUID,
+	postCutoverRetention bool,
```

Comment:
if we change the string below we don't need this new arg

Reply:
Done.